src/index.js

```diff
@@ -337,7 +337,7 @@ const Styleguide = () => {
                         Button: require('./components/Button'),
                         Checkbox: require('./components/Form/Checkbox.js'),
                         Radio: require('./components/Form/Radio.js'),
-                        Field: require('./components/Form/Field.js'),
+                        Field: require('./components/Form/Field.tsx'),
```

Comment:
Easiest is to just drop the file name ending:
```suggestion
                        Field: require('./components/Form/Field'),
```
(if you feel like it get ride of all of them in the file)